routes/web.php

```diff
@@ -477,6 +477,16 @@
         // Public access
         $router->get('/', 'UsersController@index');
         $router->get('/{id:[0-9]+}', 'UsersController@show');
+        $router->get('/settings', 'SettingsController@index');
+
+        // Sub-user routes
+        $router->group(['prefix' => '/{user_id:[0-9]+}'], function () use ($router) {
```

Comment:
@willdoran I think the routes need to be inside the hxl feature flag check ((Features::isEnabled('hxl')))

Reply:
@rowasc 
I don't think so, I think this could legitimately be considered a separate feature since we're likely to use user_settings elsewhere. Would it not deserve its own flag or no?

But I can hide it for now behind that flag with a comment.

Reply:
Everything being added for HXL (or anything else) should get a feature flag. That way we can eventually deploy it to prod but disabled till its 100% working. I would argue for just using the HXL feature flag - on the assumption that this feature flag is only going to be used to disable the feature during develop and QA, not a long lived feature flag that will be tied to users plans.